control-plane/controllers/configentries/configentry_controller_ent_test.go

```diff
@@ -210,7 +210,10 @@ func TestConfigEntryController_createsEntConfigEntry(t *testing.T) {
 
 			s := runtime.NewScheme()
 			s.AddKnownTypes(v1alpha1.GroupVersion, c.configEntryResource)
-			fakeClient := fake.NewClientBuilder().WithScheme(s).WithRuntimeObjects(c.configEntryResource).Build()
+			fakeClient := fake.NewClientBuilder().WithScheme(s).
+				WithRuntimeObjects(c.configEntryResource).
+				WithStatusSubresource(c.configEntryResource).
```

Comment:
controller-runtime now requires that you register any objects with a status subresource on your fake client for testing. You'll see this change throughout this PR.